CMakeLists.txt

```diff
@@ -177,7 +177,7 @@ if(platform_supports_ninja_browse)
 		PROPERTIES
 			OBJECT_DEPENDS "${PROJECT_BINARY_DIR}/build/browse_py.h"
 			INCLUDE_DIRECTORIES "${PROJECT_BINARY_DIR}"
-			COMPILE_DEFINITIONS NINJA_PYTHON="python"
+			COMPILE_DEFINITIONS NINJA_PYTHON="python3"
```

Comment:
Would it make sense to find the Python 3 executable via `find_package(Python3 REQUIRED COMPONENTS Interpreter)`?

Reply:
I'm not really sure, I was just hoping to do the minimum diff here

Reply:
Yes, it would. Or the "find_program" function. 

But I agree that this PR should do the minimum. A later PR can do a dynamic search.